go/libkb/track.go

```diff
@@ -375,6 +378,19 @@ func LocalTrackChainLinkFor(tracker, trackee keybase1.UID, g *GlobalContext) (re
 		g.Log.Debug("| unpack failed -> %s", err)
 		return
 	}
+
+	linkEtime := cl.GetCTime().Add(g.Env.GetLocalTrackMaxAge())
```

Comment:
Also not positive the existing creation time of the track is the right thing to use